lightsteem/client.py

```diff
@@ -30,11 +30,11 @@ def pick_id_for_request(self):
 
     def request(self, *args, **kwargs):
 
-        if kwargs.get("batch_data"):
-            # if that's a batch call, don't do any formatting on data.
-            # since it's already formatted for the app base.
-            data = kwargs.get("batch_data")
-        else:
+        # if that's a batch call, don't do any formatting on data.
+        # since it's already formatted for the app base.
+        data = kwargs.get("batch_data", None)
```

Comment:
Do we really need None here?  Also, let's use ```batch_data``` as a variable name, instead of the broad ```data```.

```
batch_data = kwargs.get("batch_data")
if not batch_data:
    # ...
```
Looks good to me if you update it that way.